mocksipipeline/instrument/optics/response.py

```diff
@@ -319,3 +323,71 @@ def get_wcs(self, observer, roll_angle=90 * u.deg):
             pc_matrix=pc_matrix,
             observer=observer,
         )
+
+    def psf(self, wavelength=None, mask_oversample=4, fft_oversample = 4):
```

Comment:
Is there a reference for implementing the PSF calculation in this way? I just don't want to forget where these expressions came from.